netmap/node_info.go

```diff
@@ -460,6 +460,40 @@ func (x NodeInfo) IterateAttributes(f func(key, value string)) {
 	}
 }
 
+// GetAttributes returns all the node attributes.
+// Each attribute is a [2]string slice: {"key", "value"}.
+//
+// See also Attribute, IterateAttributes.
+func (x NodeInfo) GetAttributes() [][2]string {
+	attrs := make([][2]string, len(x.m.GetAttributes()))
+	for i, attr := range x.m.GetAttributes() {
+		attrs[i] = [2]string{attr.GetKey(), attr.GetValue()}
+	}
+	return attrs
+}
+
+// SetAttributes sets list of node attributes.
+// Each attribute is a [2]string slice: {"key", "value"}.
+// Both key and value of attributes MUST NOT be empty.
+//
+// See also SetAttribute.
+func (x *NodeInfo) SetAttributes(attrs ...[2]string) {
+	netmapAttrs := make([]netmap.Attribute, 0, len(attrs))
+	for _, attr := range attrs {
+		if attr[0] == "" {
+			panic("empty key in SetAttributes")
+		} else if attr[1] == "" {
```

Comment:
i would not add `else` here

Reply:
Can you explain why? I took as an example from here:
https://github.com/nspcc-dev/neofs-sdk-go/blob/ece099d18f79c27b446efba6e24efa8574a6c474/netmap/node_info.go#L465-L470

Reply:
Ah, I understood, only the word `else` should not be added.

Reply:
it increases cognitive load imo. we have already touched it, i perceive this code as this (and i will always attach this link in such threads): https://www.refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html